integreat_cms/static/src/js/forms/icon-field.tsx

```diff
@@ -6,7 +6,7 @@ import SelectMedia from "../media-management/select-media";
 
 // Set event handlers
 window.addEventListener("load", () => {
-    const iconField = document.getElementById("id_icon");
+    const iconField = document.getElementById("poi-category-icon-field");
```

Comment:
Refers not to this line, but to this file:
Change of the icon produces TypeError
![image](https://user-images.githubusercontent.com/115008338/214591759-2df735a7-ca17-4557-8b42-27e9144d6393.png)

Reply:
indeed, this icon field is different from the `poi-category-icon-field`, so please rename it back to `id_icon`
```suggestion
    const iconField = document.getElementById("id_icon");
```

Reply:
Applied!